x-pack/solutions/search/plugins/search_solution/search_navigation/public/classic_navigation.test.ts

```diff
@@ -153,20 +135,20 @@ describe('classicNavigationFactory', function () {
   it('returns name if provided over the deeplink title', () => {
     const items: ClassicNavItem[] = [
       {
-        id: 'searchIndices',
+        id: 'searchConnectors',
         deepLink: {
-          link: 'enterpriseSearchContent:searchIndices',
+          link: 'enterpriseSearchContent:connectors',
         },
-        name: 'Index Management',
+        name: 'Connectors',
```

Comment:
`Connectors` is the title of the `navlink` for `enterpriseSearchContent:connectors` (see line 23 where we mock nav links).

So we should change this name to ensure this unit test continues to test the behavior here, or you could use a different deep link. 

for example this could be 
```suggestion
        name: 'Date connectors',
```

the important thing is that we're testing that you can override the deeplink title if you want to when defining side nav items using deep links (navlink). So for this unit test `name` and the `title` on the mocked deeplink should be different.